JamfSync/Model/JamfProPackageClassicApi.swift

```diff
@@ -17,6 +17,7 @@ class JamfProPackageClassicApi: JamfProPackageApi {
             if let jsonPackages = try? decoder.decode(JsonCapiPackages.self, from: data) {
                 packages.removeAll()
                 for package in jsonPackages.packages {
+                    print("[JamfProPackageClassicApi] \(package.name)")
```

Comment:
This should be a debug log message (assuming it will be useful in the future) or removed.

Reply:
removed.